module-2/backend/week_one.md

```diff
@@ -5,30 +5,105 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+* DELETE: delete , GET: render a view, POST: submit content, PUT: update content, PATCH: sililar to PUT, used in Rails.
 2. What is Sinatra?
+
+* Sinatra is a web-application framework. It allows the developer to configure things to his liking more than Rails.
 4. What is MVC?
+
+* Model, View, Controller. Model: holds our logic; Controller: serves as our router; View: displays content
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+* To be in sync with what Rails does? 

```

Comment:
To some degree this is definitely true. We're using Sinatra as a stepping stone. I think that we were thinking about this in terms of conventions in a more general sense, which allow us to step into an unfamiliar code base and quickly get up to speed so we can contribute (and allows others to do the same in our code base).